qbosdk/apis/api_base.py

```diff
@@ -259,7 +259,7 @@ def _post_request(self, data, api_url):
             result = json.loads(response.text)
             return result
     
-        logger.info('Payload for post request: %s', data)
+        logger.debug('Payload for post request: %s', data)
```

Comment:
this should be before 257 block, we don't end up reaching here since we're returning at 260